charts/cloudquery/templates/cronjob.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   {{- include "cloudquery.labels" . | nindent 4 }}
 spec:
   schedule: "{{ .Values.schedule }}"
+  successfulJobsHistoryLimit: "{{ .Values.cronJobLimit }}"
+  failedJobsHistoryLimit: "{{ .Values.cronJobFailedJobsLimit }}"
```

Comment:
Remove the `""`, I think :)

When running on my machine:
```
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(CronJob.spec.failedJobsHistoryLimit): invalid type for io.k8s.api.batch.v1.CronJobSpec.failedJobsHistoryLimit: got "string", expected "integer", ValidationError(CronJob.spec.successfulJobsHistoryLimit): invalid type for io.k8s.api.batch.v1.CronJobSpec.successfulJobsHistoryLimit: got "string", expected "integer"]
```

Reply:
Whoops, dunno how I missed that 🙃 

Pushed up [382f4ee](https://github.com/cloudquery/helm-charts/pull/201/commits/382f4ee8d67c13102374d6843b2a44faa1c4039e) lmk if that doesn't resolve the issue.